SI/Sem.05/Pract.05/Task02.cpp

```diff
@@ -0,0 +1,56 @@
+#include <iostream>
+using namespace std;
+void swap(int& a, int& b)
+{
+	int temp = a;
+	a = b;
+	b = temp;
+}
+void SortEvenNumsAscending(int* arr, int Size)
+{
+	for (int i = 0; i < Size; i++)
+	{
+		for (int j = i + 1; j < Size; j++)
+		{
+			if (arr[i] % 2 == 0 && arr[i] > arr[j])
+			{
+				swap(arr[i], arr[j]);
+			}
+		}
+	}
+}
+void SortOddNumsDescending(int* arr, int Size)
+{
+	for (int i = 0; i < Size; i++)
+	{
+		for (int j = i + 1; j < Size; j++)
+		{
+			if (arr[i] % 2 != 0 && arr[i] < arr[j])
+			{
+				swap(arr[i], arr[j]);
+			}
+		}
+	}
+}
```

Comment:
This solutions works. Maybe it will be more optimal to first put the even numbers to the left and the odd to the right and then sort every half? @Justsvetoslavov